cmd/mesh_bridge/main.go

```diff
@@ -0,0 +1,106 @@
+// +build !js
+
+// mesh_bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+type clientEnvVars struct {
+	FirstWSRPCAddress  string `envvar:"FIRST_WS_RPC_ADDRESS"`
+	SecondWSRPCAddress string `envvar:"SECOND_WS_RPC_ADDRESS"`
+	Verbose            bool   `envvar:"VERBOSE"`
+}
+
+func main() {
+	env := clientEnvVars{}
+	if err := envvar.Parse(&env); err != nil {
+		panic(err)
+	}
+
+	if env.Verbose {
+		log.SetLevel(log.Level(4))
+	} else {
+		log.SetLevel(log.Level(0))
+	}
+
+	firstClient, err := rpc.NewClient(env.SecondWSRPCAddress)
```

Comment:
I think this would be clearer if the `firstClient` used the `FirstWSRPCAddress` and the `secondClient` used the `SecondWSRPCAddress`. Anything I'm missing?

Reply:
Even better would be `Version8WSRPCAddress`, `v8Client`, etc.